### DISCUSSION TITLE
Open Graph について

### DISCUSSION THREAD

**Discussion by blono** [increments/qiita-discussions#37] [要望・アイデア・不具合]
例えばですが、記事が以下の 1 行のみの場合

> https://www.google.co.jp/

Open Graph が働きますが、記事が次の 2 行のみの場合

> https://www.google.co.jp/
> aaa

Open Graph が働かなくなります。
Open Graph を制御する方法とかあったら教えて欲しいです！

**Comment by daikisuyama** [comment]
改行によって働くようになると思いますが、それでは不十分でしょうか？

**Reply by uhooi** [reply]
私も同意見です。
ただ「上下に改行を入れた URL」をそのまま表示する方法はないかもしれないので、どうしても実現したいなら制御が必要かもしれません。

**Comment by kskwtnk** [comment]
@blono ご意見ありがとうございます！
@daikisuyama @uhooi おふたりも、補足ありがとうございます！

既に説明いただいているように「OGPのimageを表示するには改行を必要とする」作りで提供しています
文中に単なるリンクとして展開する場合もありつつ、OGPとして表示したい場合もありつつなので、こちらは改行が必要なままとさせていただきたいです 🙏

**Reply by blono** [reply]
改行があれば行けるんですね！
個人的にはそれでも良さそうに思えました、教えて頂きありがとうございます！
